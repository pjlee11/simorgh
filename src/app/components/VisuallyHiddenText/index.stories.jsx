import React from 'react';
import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
import VisuallyHiddenText from './index';

storiesOf('VisuallyHiddenText', module).add('default', () => (
  <VisuallyHiddenText>Some offscreen text</VisuallyHiddenText>
));
