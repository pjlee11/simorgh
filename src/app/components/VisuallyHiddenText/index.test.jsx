import React from 'react';
import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
import VisuallyHiddenText from './index';

describe('VisuallyHiddenText', () => {
  shouldMatchSnapshot(
    'should render off screen text for screen readers',
    <VisuallyHiddenText>Some offscreen text</VisuallyHiddenText>,
  );
});
