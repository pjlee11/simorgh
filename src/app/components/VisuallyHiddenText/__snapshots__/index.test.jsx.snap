// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisuallyHiddenText should render off screen text for screen readers 1`] = `
.c0 {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  border: 0;
  overflow: hidden;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  top: 0;
}

<span
  className="c0"
>
  Some offscreen text
</span>
`;
